bluepysnap/nodes.py

```diff
@@ -436,8 +649,10 @@ def ids(self, group=None, limit=None, sample=None):
             result = utils.ensure_list(group)
             self._check_ids(result)
             preserve_order = isinstance(group, collections.Sequence)
+
         if sample is not None:
             if len(result) > 0:
+                sample = sample if sample <= len(result) else len(result)
```

Comment:
You don't like `min` don't you ? :)

Reply:
lol this is my classic tunnel vision when I code